### PR TITLE
PT-223 | Venue info

### PR DESCRIPTION
**Pull request by jorilindell**
**Description**
Show event location as default place for an occurrence
Implement reusable VenueInfo component to show venue info

**Closes**
PT-223

### CODE REVIEW

File: src/icons/IconFood.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = { className?: string };
+
+export default function IconDelete({ className = '' }: Props) {
```
Comment (badrange) [inline src/icons/IconFood.tsx:5]: IconDelete? The file name is IconFood.tsx

Also, have you noticed that there are lots of new icons in HDS now?

File: src/icons/IconGarderobe.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = { className?: string };
+
+export default function IconDelete({ className = '' }: Props) {
```
Comment (badrange) [inline src/icons/IconGarderobe.tsx:5]: IconDelete? The file name is IconGarderobe..

Comment (jorilindell) [reply src/icons/IconGarderobe.tsx:5]: My bad. Copy pasted to skeleton for it. Didn't find this one from HDS so I had to create it myself.

File: src/icons/IconFood.tsx (hunk 1 above)
Comment (jorilindell) [reply src/icons/IconFood.tsx:5]: My bad. Copy pasted to skeleton for it. Didn't find this one from HDS so I had to create it myself.

**Pull request merged**